### PR TITLE
Apply theme to curate 

### LINKED ISSUE #38: UI Layout Update
**Is your feature request related to a problem? Please describe.**
The interface design requires updating to reflect gliff'ai's new visual identity but also to improve the UI and UX for users.

**Describe the solution you'd like**
I've created the UI design based on the annotate's interface style, the current features and the team's 'wants'/ideas for the curate interface.

The below images showcase the grid at it's three sizes, let's call them small, medium, large. The small grid is 11 images wide, the medium grid is 7 images wide and the large grid is 5 wide. I have given them a tight 8px gap to minimise The Hermann grid illusion where, optically, dots appear in the cross over space between images. 

The user changes the size via the L M S icons within their own box at the top-left. I've designed this around using the natural scroll bar of the internet browser. The scrolling should only affect the image grid. Images below show the beginning and scroll state. The images get cut off before the tools along the top.

Large Grid.
Curate Layout -  large images
Curate Layout -  large images   example scroll

Medium Grid.
Curate Layout -  medium images
Curate Layout -  medium images   example scroll

Small Grid.
Curate Layout -  small images
Curate Layout -  small images   example scroll

The group of collapsable/drop-down menus on the left is where users should be able to access their search filters. I've designed the ability to retain and remove multiple search filters. 

Curate Layout -  search filters   not active

Below is the annotation labels menu. I would anticipate similar visuals for the image tags and team members (although this one could use user profile pictures instead of a label icon?). I have also showed the hover state for the 'select all labels' and 'deselect all labels' button. Also I discussed with Silvia how if a user was to double click on a label it would keep selected only that label and deselect all the other and vice versa (https://github.com/gliff-ai/curate/issues/31).

Curate Layout -  annotation labels 0 active
Curate Layout -  annotation labels select all  hover state
Curate Layout -  annotation labels select all  hover state  – 1

The active search filters came from another discussion with Chas (https://github.com/gliff-ai/curate/issues/30) about showing the user their active search filters. Within this I have moved the search bar to create the filters from the top bar down. Named the first category 'search category' to simplify for people like me who aren't aware of such technical words. I wondered if instead of the second box being 'key or key value' that it was '...' until a 'search category' was picked which can then add a better word into that search field? The icon to the left is the search filter offering the filter types like alphabetical order (I am not too sure the full list we would want to offer?) within a drop-down list.

Curate Layout -  search filters

**Describe alternatives you've considered**
I have experimented around with the sizing of the image grids to fit which ones would be most useful before concluding on the size of the grids. I also have experimented with utilising the top bar but in the full product that top bar needs to be reserved for other establishment content.
I am open to any thoughts or suggestions design/user experience wise.

**Additional context**
Adobe XD link: [https://xd.adobe.com/view/f7272ef3-f631-4ed9-af1f-f737f5148926-a274/](url)
[SVG files of plain icons are in UI/UX Google Drive folder]

### LINKED ISSUE #89: UX design for whole-image labelling
**Is your feature request related to a problem? Please describe.**
We need a design for whole-image labelling in CURATE. We should be able to add and delete labels to each thumbnail image. See also #49.  

**Describe the solution you'd like**
?

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### LINKED ISSUE #101: Search UI Dropdown
**Is your feature request related to a problem? Please describe.**
The dropdown menu of the search fields needs a UI design that remains within the same styling of the gliff products.

**Describe the solution you'd like**
The dropdown menu is very much like the styling of a dropdown menu in ANNOTATE.
The dropdown is almost the same width but a little shorter. This allows it to cover over enough that it isn't hard to read or see the menu. Text on a plain background, but when hovered should be a bar of green behind.

Curate Layout -  medium images  – w- search category dropdown

**Describe alternatives you've considered**
N/A

**Additional context**
https://xd.adobe.com/view/ff4a6f10-184f-48ef-9085-33fbfa12b7c4-aa19/

### PR DESCRIPTION
**Pull request by ChrisBaidoo**
# Description

Apply gliff.ai  new branding guide to curate.

closes  #101 
closes  #38 
closes  #89

# Dependency changes
   added:  "react-inlinesvg": "^2.3.0"

# Testing

N/A 
# Documentation

N/A 

# Migrations (if applicable)
N/A 

Have any database migrations been committed as part of this pull request?
N/A 

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] New migrations have been committed
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by cooper667** [comment]
Could the buttons have tooltips? (I have no idea what the one on the bottom left should do)

File: examples/index.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
     />
   </head>
 
-  <body style="padding: 0; margin: 0">
+  <body style="padding: 0; margin: 0; background-color: #F2F2F2">
     <div id="react-container" style="height: 100%"></div>
   </body>
```
Comment (SilviaZeta) [inline examples/index.html:20]: Can we move this styling out of the html file, pls?

File: src/components/SizeThumbnails.tsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+export default function SizeThumbnails(props: Props): ReactElement {
+  const classes = useStyles();
+  const [buttonClicked, setButtonClicked] = useState("");
+
+  useEffect(() => {
+    if (buttonClicked === "Large") {
+      props.largeThumbnails();
+    }
+    if (buttonClicked === "Medium") {
```
Comment (SilviaZeta) [inline src/components/SizeThumbnails.tsx:56]: else if?

**Comment by SilviaZeta** [comment]
When I click on the select images button, then on a thumbnail, the only thing left in the left section is the delete button and I don't find a way to return to the original view.

**Review by cooper667 — DISMISSED** [review]

**Review by ChasNelson1990 — CHANGES_REQUESTED** [review]
Looking good guys!

A few visual things from me:

- [ ] the small/medium/large icons are, on my screen a little aliased. Not sure if this screenshot shows it well enough but it looks a bit naff because of that. Can we just check we're using the SVGs?
Screenshot_2021-06-24_10-30-24
- [x] What does this icon mean - didn't seem to do anything when I clicked. I think @cooper667 suggested tooltips - good suggestion, can we add them?
Screenshot_2021-06-24_10-31-12
- [ ] Are the Select All/Select None labels means to hover highlight in this way (not fully coloured but partly coloured)? I note that they are the only buttons to have a hover effect, shouldn't all the buttons hover the same as in ANNOTATE?
Screenshot_2021-06-24_10-31-38
- [ ] Have you checked this out on an iPad or even just a 50% width desktop monitor? Everything is a bit skewiff, maybe @joshuajames-smith could have a think about how to deal with these smaller screens?
salmon-meadow-05ee3f103-112 westeurope azurestaticapps net_(iPad)

**Review by cooper667 — APPROVED** [review]

**Review by cooper667 — APPROVED** [review]

**Review by SilviaZeta — APPROVED** [review]

**Review by philipjackson — APPROVED** [review]

**Review by ChasNelson1990 — APPROVED** [review]

**Pull request merged**